econml/metalearners.py

```diff
@@ -150,39 +149,42 @@ def fit(self, Y, T, X, inference=None):
         self : an instance of self.
         """
         # Check inputs
+        if X is None:
+            X = np.ones((Y.shape[0], 1))
```

Comment:
[minor]
I think that in this case, the default could be a 0-column array rather than a column of ones (the columns from T will still be there):

```suggestion
            X = np.empty((Y.shape[0], 0))
```

Reply:
On the other hand, maybe it's silly to even allow X=None because there is no W (unlike DML)

Reply:
Yes. In the setting of Slearner, X = None is the same with learning the diff of mean(Y) in each class. I will keep it for now.